pkg/document/time/version_vector.go

```diff
@@ -97,9 +113,102 @@ func (v VersionVector) AfterOrEqual(other VersionVector) bool {
 	return true
 }
 
+// EqualToOrAfter returns whether this VersionVector's every field is equal or after than given ticket.
+func (v VersionVector) EqualToOrAfter(other *Ticket) bool {
+	return v[other.actorID.bytes] >= other.lamport
+}
+
 // After returns whether this VersionVector is causally after the given ticket.
+// TODO(JOOHOJANG) check below function is necessary
```

Comment:
Did we need this function? If we didn't need it, I think we could delete it.

```suggestion
// TODO(JOOHOJANG): check below function is necessary
```